app/src/main/java/com/github/fribourgsdp/radio/GameActivity.kt

```diff
@@ -184,4 +189,42 @@ open class GameActivity : AppCompatActivity(), GameView {
         voiceChannel.enableAudioVolumeIndication(200,3,true)
         voiceChannel.joinChannel(voiceChannel.getToken(userId, gameUid.toString()), gameUid.toString(), "", userId)
     }
+
+    override fun displayLyrics(lyrics : String) {
+        showLyricsButton.visibility = View.VISIBLE
+        showLyricsButton.setOnClickListener { displayLyrics(lyrics) }
+        val lyricsPopup = LyricsPopup(lyrics)
+        lyricsPopup.show(supportFragmentManager, "lyricsPopup")
+
+//        val lyricsDialog = Dialog(this)
+//        lyricsDialog.requestWindowFeature(Window.FEATURE_NO_TITLE)
+//        lyricsDialog.setContentView(R.layout.popup_lyrics)
+//        lyricsDialog.setCanceledOnTouchOutside(false)
+//        lyricsDialog.findViewById<TextView>(R.id.lyricsPopupTextView).text = lyrics
+//        lyricsDialog.findViewById<ImageView>(R.id.close_popup_button).setOnClickListener{
+//            lyricsDialog.dismiss()
+//        }
+//        lyricsDialog.show()
+
+//        val inflater = getSystemService(LAYOUT_INFLATER_SERVICE) as LayoutInflater
+//        val popupView: View = inflater.inflate(R.layout.popup_lyrics, null)
+//        val lyricsTextView : TextView = popupView.findViewById(R.id.lyricsPopupTextView)
+//        val width = LinearLayout.LayoutParams.WRAP_CONTENT
+//        val height = LinearLayout.LayoutParams.WRAP_CONTENT
+//        val focusable = true // lets taps outside the popup also dismiss it
+//
+//        lyricsPopup = PopupWindow(popupView, width, height, focusable)
+//        lyricsPopup.showAtLocation(songTextView, Gravity.CENTER, 0, 0);
+//        lyricsTextView.text = lyrics
+//
+////        // dismiss the popup window when touched
+//        popupView.findViewById<ImageView>(R.id.close_popup_button).setOnClickListener{
+//            lyricsPopup.dismiss()
+//        }
+//        popupView.setOnTouchListener { v, _ ->
+//            v.performClick()
+//            lyricsPopup.dismiss()
+//            true
+//        }
```

Comment:
is there a reason for keeping this in the PR? I think since we have versioning, this could be removed

Reply:
Oh you're right, I can be oblivious sometimes when it comes to comments and debug messages, I shall remove it.